api/nutritionix.py

```diff
@@ -1,6 +1,26 @@
-# drinks (?)
+import requests
+import os
+from pprint import pprint
+#Importing requests, os and pprints
 
-# talk to to the api, return drink info
+# Application KEY to run program 
+app_key = os.environ.get('DRINK_KEY')
+#Aplication ID to run the program
+app_id = os.environ.get('DRINK_ID')
 
-def get_drink(search_term):
-    return 'coke'
\ No newline at end of file
+# User will enter the drink they wanna search 
+search_drink = 'COFFEE'
```

Comment:
I think line 12 here search_drink = "COFFEE"  can be removed because def get_drink(search_term) would've provided the search term (from the user). Hope that makes sense. Let me know if you have any questions.

Reply:
It does make sense, just update it

Reply:
@jx4515ga The user will be prompt for an input from ui.py example: drink_name = input("please enter a drink name: "). Whatever the user types in will be passed to the controller then to nutritionix def get_drink(search_term). The api would then return the search results. I believe you don't can completely take out line 12. It might help to look at spoonacular or at least that's how I think it's processing the data. I referenced this off of Clara's example.